demos/demo36_experimental_mcmc_module.py

```diff
@@ -77,5 +80,48 @@
 
 # This stored samples in chunks of 100 in the folder demo36_sampler_samples
 
+# %% ULA
```

Comment:
I wonder if we need to add all these explicit examples inthe demo file? There are some bugs that we will fix later like the need for burnthin(1). I would suggest we actually remove most of these lines and rely primarily on the unit tests.

If you want you can leave a smalle xample of the ULA and MALA samplers.

Reply:
Remove these unnecessary examples as suggested